### PR TITLE
Omit wrapping tokens and control groups from client counts

### PR DESCRIPTION
**Pull request by briankassouf**

### CODE REVIEW

File: vault/activity_log_test.go [hunk 1]
```diff
@@ -91,6 +91,54 @@ func TestActivityLog_Creation(t *testing.T) {
...
+		Path:         "test",
+		Policies:     []string{controlGroupPolicyName},
+		CreationTime: time.Now().Unix(),
+		TTL:          3600,
+		NamespaceID:  namespace_id,
+	})
+
+	a.fragmentLock.Lock()
+	if a.fragment != nil {
+		t.Fatal("fragment created")
```
Comment (swayne275) [inline vault/activity_log_test.go:137]:
```suggestion
		t.Fatal("fragment not created")
```

**Review by swayne275 — APPROVED** [review]
I'm approving because I'll be heading out for a flight soon, but i think that code suggestion i added needs to go in first. other than that, looks good!

File: vault/activity_log_test.go (hunk 1 above)
Comment (swayne275) [reply vault/activity_log_test.go:137]: wait a sec, do we want the fragment to be created here for non-wrapped token? if the `a.fragment != nil` check is correct, ignore my suggestion!

Comment (briankassouf) [reply vault/activity_log_test.go:137]: Yeah we do not want it to be created here since this is a control group token

**Pull request merged**